conf/api.routes

```diff
@@ -1,2 +1,2 @@
-GET         /api/address/:id                      controllers.ImporterAddressController.onLoad(id: String)
+GET         /api/eoriDetails/:id                  controllers.EoriDetailsController.onLoad(id: String)
```

Comment:
Please change the route to `/subscriptions/subscriptiondisplay/v1` so it matches the route exposed by EIS. This will then simply become a config change to re-point to the real endpoint
```suggestion
GET         /subscriptions/subscriptiondisplay/v1                  controllers.EoriDetailsController.onLoad(regime: String, acknowledgementReference: String, EORI: String)
```

Reply:
The actual endpoint expects the additional querystring parameters Example:
`http://<host>:<port>/subscriptions/subscriptiondisplay/v1?regime=CDS&acknowledgementReference=11a2b17559e64b14be257a112a7d9e8e&EORI=GB123456789`